.github/workflows/build.yml

```
@@ -151,13 +151,23 @@ jobs:
     if: fromJSON(needs.check_source.outputs.run_tests)
     strategy:
       matrix:
+        os:
+          - windows-latest
         arch:
-        - Win32
-        - x64
-        - arm64
+          - x64
         free-threading:
-        - false
-        - true
+          - false
+          - true
+        include:
+          - os: windows-aarch64
+            arch: arm64
+            free-threading: false
+          - os: windows-aarch64
+            arch: arm64
+            free-threading: true
+          - os: windows-latest
+            arch: Win32
+            free-threading: false
     uses: ./.github/workflows/reusable-windows.yml
     with:
       arch: ${{ matrix.arch }}
```

As close as I can get with the diff view:

```suggestion
            free-threading: false
    uses: ./.github/workflows/reusable-windows.yml
    with:
      arch: ${{ matrix.arch }}
      os: ${{ matrix.os }}
```